var/spack/repos/builtin/packages/libsharp/package.py

```diff
@@ -0,0 +1,60 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
+from spack import *
+
+
+class Libsharp(AutotoolsPackage):
+    """Libsharp is a code library for spherical harmonic transforms (SHTs) and
+    spin-weighted spherical harmonic transforms, which evolved from the libpsht
+    library."""
+
+    variant('openmp', default=True, description='Build with openmp support')
+    variant('mpi', default=True, description='Build with MPI support')
+
+    homepage = "https://github.com/Libsharp/libsharp"
+    git      = "https://github.com/Libsharp/libsharp.git"
+
+    version('1.0', commit='593d4eba67d61827191c32fb94bf235cb31205e1')
```

Comment:
this should probably be a date based version since releases aren't tagged (i.e. 2018-01-17)

Reply:
I've also just put in a request for having libsharp receive some version: https://github.com/Libsharp/libsharp/issues/16

Reply:
Version changed to date. Thanks!